### PR TITLE
Update Contributing “code-of-conduct”

### PR DESCRIPTION
**Pull request by tomoyanonymous**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/en/docs/Contributing/code-of-conduct.en.md [hunk 1]
```diff
@@ -2,8 +2,85 @@
...
+
+## The mimium community strives to be:
+
+- Prioritize Diversity. We strive to be a community that welcomes and supports people of all backgrounds and identities.
+- Considerate: Your work will be used by other people, and you in turn will depend on the work of others. Any decision you take will affect users and colleagues, and you should take those consequences into account when making decisions. 
+- Respect other communities. mimium has been made under the influence of many preceding languages, projects, and music/sound artworks. mimium is not made for surpassing or making them obsolete. We strive to grow our community and other various communities got involved with us to be more valuable through repetetive communication.
+
+## Expected Behavior as a member of the community
+
+- Participate in an authentic and active way. In doing so, you contribute to the - health and longevity of this community.
```
Comment (yutannihilation) [inline content/en/docs/Contributing/code-of-conduct.en.md]:
```suggestion
- Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
```

File: content/en/docs/Contributing/code-of-conduct.ja.md [hunk 2]
```diff
@@ -1,7 +1,88 @@
...
+コミュニティメンバーが容認できない行動をした場合、コミュニティ主催者は警告なし（有料イベントの場合は返金もなし）での一時的な禁止またはコミュニティからの永久追放も含め、妥当と判断される措置を講じることができる。
+
+## 容認できない行動を目撃したり受けたりした場合
+
+容認できない行動を受けたり目撃したりした場合、または他の懸念がある場合、すぐにコミュニティ主催者に知らせてください。
+コミュニティの主催者は、コミュニティのメンバーが地元の法執行機関に関わること、または容認できない行動を経験した人が安心感を得ることを最大限支援します。主催者は行動規範に沿わないコメント、コード、issueやその他のコントリビューションを削除、編集、拒否する権利と義務を持っています。
+
+## 苦情の対応について
+
+本行動規範の違反のぬれぎぬを着せられたと感じる場合、苦情に関して簡潔に説明することによりイベント主催者の一人に知らせるべきです。あなたの苦情は既存の管理ポリシーに従って処理されるでしょう。
```
Comment (yutannihilation) [inline content/en/docs/Contributing/code-of-conduct.ja.md]:
```suggestion
本行動規範の違反のぬれぎぬを着せられたと感じる場合、苦情に関して簡潔に説明することによりコミュニティ主催者の一人に知らせるべきです。あなたの苦情は既存の管理ポリシーに従って処理されるでしょう。
```

File: content/en/docs/Contributing/code-of-conduct.en.md [hunk 3]
```diff
@@ -2,8 +2,85 @@
...
+If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event).
+
+## If You Witness or Are Subject to Unacceptable Behavior
+
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer as soon as possible. You can find a list of organizers to contact for each of the supporters of this code of conduct at the bottom of this page. Additionally, community organizers are available to help community members engage with local law enforcement or to otherwise help those experiencing unacceptable behavior feel safe. In the context of in-person events, organizers will also provide escorts as desired by the person experiencing distress.
+
+
+## Addressing Grievances
+
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify one of the event organizers with a concise description of your grievance. Your grievance will be handled in accordance with our existing governing policies.
```
Comment (yutannihilation) [inline content/en/docs/Contributing/code-of-conduct.en.md]:
```suggestion
If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify one of the community organizers with a concise description of your grievance. Your grievance will be handled in accordance with our existing governing policies.
```

**Review by yutannihilation — COMMENTED** [review]
Minor typos.

**Comment by tomoyanonymous** [comment]
Thanks for reviewing! Committed proposed changes.

**Comment by yutannihilation** [comment]
Thanks!

**Pull request merged**